deploy/template.yml

```diff
@@ -569,7 +569,7 @@ parameters:
   value: bop
 - name: AUTH_GATEWAY_URL
   required: true
-  value: "http://apicast.3scale-staging.svc.cluster.local:8890/internal/certauth"
```

Comment:
I noticed that this URL references `staging` not `stage`. Need to confirm this should be pointing to the Stage Cluster (`crcs02ue1`).

Reply:
Cloud-Connector should be pointed at 3scale running in stage.  Yeah, I think this should be the stage cluster.